quick_install.sh

```diff
@@ -81,9 +81,10 @@ function infer_platform() {
 
 function __bliman_quick_install() {
 
-	local force_flag
-	force_flag=$1
-
+	local genesis_path force_flag
+        genesis_path=$1	
```

Comment:
Do we have to pass the genesis file path. We can fix it in home directory. Make it as an env variable. If the user wishes to change it he can export it.

Reply:
The path is not mandatory the default locations would be /$HOME/.bliman or Present working directory. if not found at these locations than it will check for the path provided.